File: Spring Boot Application/src/main/java/com/application/Bank_Application_SpringBoot/Services/CustomerService.java
package com.application.Bank_Application_SpringBoot.Services;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

import com.application.Bank_Application_SpringBoot.Model.Customer;
import com.application.Bank_Application_SpringBoot.Repositories.CustomerRepository;

@Service
public class CustomerService {
	
	@Autowired
	private CustomerRepository customerRepository;
	
	public Customer getCustomerById(int id) {
		try {
			return customerRepository.findById(id).orElse(null);
		}catch(Exception err) {
			return null;
		}
	}
	
    public final int createCustomer(String firstName, String lastName, String address, String email, int password){

    	try {
    		Customer newCustomer = new Customer( firstName, lastName, email, address, password);
        	customerRepository.save(newCustomer);
            return newCustomer.getId();
    	}catch(Exception err) {
    		return -1;
    	}
    }
 
	 public final boolean loginCustomer(int customerID, int password){
	
	 	try {
	 		Customer currentCustomer = getCustomerById(customerID);
	 		if(currentCustomer == null || currentCustomer.getPassword() != password)return false;
	 		return true;
	 	}catch (Exception err) {
	 		return false;
		}
	 }
}

File: Node_Express_Sequelize/Services/CustomerService.js
const { getConnection, closeConnection } = require('../Utils/SequelizeConnectionUtil');

const createCustomer = async(first_name, last_name, address, email, password) =>{
    const [seq, models] = await getConnection();
    if(seq != null && models != null){
        const insertId = models.customer.create({
            first_name,
            last_name,
            email,
            address,
            password
        }).then((data)=>{
            closeConnection(seq);
            return data.id;
        }).catch(err => {
            closeConnection(seq);
            return -1;
        })
        return insertId;
    }
    else if(seq != null)closeConnection(seq);  
    return -1;
}

const loginCustomer = async(id, password) => {
    const [seq, models] = await getConnection();
    if(seq != null && models != null){
        const data = models.customer.findAll({
           where: {
                id,
                password
            }
        }).then((data)=>{
            closeConnection(seq);
            return data;
        }).catch(err => {
            closeConnection(seq);
            return [];
        })
        return data;
    }
    else if(seq != null)closeConnection(seq);  
    return [];
}

module.exports = {
    createCustomer,
    loginCustomer
} 